orangecontrib/network/widgets/graphview.py

```diff
@@ -301,7 +301,8 @@ def mouseMoveEvent(self, event):
         if not self._clicked_node:
             for node in self._selection: node.setSelected(True)
         if not self._pressed:
-            self.setCursor(Qt.OpenHandCursor if self.itemAt(event.pos()) else Qt.ArrowCursor)
+            is_node = self.itemAt(event.pos()) and isinstance(self.itemAt(event.pos()), Node)
+            self.setCursor(Qt.OpenHandCursor if is_node else Qt.ArrowCursor)
```

Comment:
Hmm, could we not just change the cursor in `mousePressEvent`, then change it back in `mouseReleaseEvent`, leaving it as it is in `mouseMoveEvent`?

Reply:
Thanks. I found two more bugs.